draft-ietf-quic-qpack.md

```diff
@@ -127,6 +127,16 @@ The dynamic table consists of a list of header fields maintained in first-in,
 first-out order.  The dynamic table is initially empty.  Entries are added by
 instructions on the encoder stream (see {{encoder-stream}}).
 
+The maximum size of the dynamic table can be modified by the encoder, subject to
+a decoder-controlled limit (see {{configuration}} and {{size-update}}).  The
+initial maximum size is the applicable value of this limit when HTTP requests or
+responses are first permitted to be sent. For clients using 0-RTT data in
```

Comment:
It might be enough to say that "the initial maximum is determined by the corresponding setting".